message/messaging_service.hh

```diff
@@ -274,7 +274,7 @@ private:
 private:
     config _cfg;
     // map: Node broadcast address -> Node internal IP for communication within the same data center
-    std::unordered_map<gms::inet_address, gms::inet_address> _preferred_ip_cache;
+    std::unordered_map<gms::inet_address, gms::inet_address> _preferred_ip_cache, _preferred_to_endpoint;
```

Comment:
The comments is wrong for _preferred_to_endpoint. Let's put it on a new line and add correct comments.

Reply:
Original discussion is here here: https://github.com/scylladb/scylla/pull/9714#discussion_r764072237

Reply:
I have changed the comment to indicate that the second member is the reversed mapping.